bin/cmd/docker

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env -S pkgx +docker bash
+
+set -eo pipefail
+
+if [ ! -d "$PKGX_PANTRY_PATH" ]; then
+  echo "error: \`brewkit docker\` can only be run from a pantry checkout" >&2
+  exit 64
+fi
+
+d="$(cd "$(dirname "$0")/../.." && pwd)"
+
+if [ -z "$GITHUB_TOKEN" ]; then
+  GITHUB_TOKEN=$(pkgx gh auth token)
+fi
+
+if [ "$1" = --pull ]; then
+  docker image pull pkgxdev/pkgx
+  shift
+fi
+
+if [ -z "$1" ]; then
+  echo "error: missing command" >&2
+  exit 64
+fi
+
+CMD=$1
+shift
+
+mkdir -p "$PKGX_PANTRY_PATH/prefixes/linux"
+
+exec docker run \
+  --name brewkit.pkgx.dev \
+  --rm \
+  --tty $PULL \
```

Comment:
is this variable unused or am i missing it? or is it just snuck in there to make life easy for people who use this a lot?